homeassistant/components/brother/sensor.py

```diff
@@ -120,15 +123,27 @@ def unit_of_measurement(self):
     @property
     def available(self):
         """Return True if entity is available."""
-        return self.printer.available
+        return self.coordinator.last_update_success
+
+    @property
+    def should_poll(self):
+        """Return the polling requirement of the entity."""
+        return False
 
     @property
     def device_info(self):
         """Return the device info."""
         return self._device_info
 
-    async def async_update(self):
```

Comment:
Normally we want to keep `async_update` and have it await `coordinator.async_request_refresh` to allow the user to schedule manual updates via the common entity update service.

Reply:
@MartinHjelmare Do I need to add new PR or commit changes to this PR to resolve your comments?

Reply:
As this PR is merged a new PR is required. Please make a new branch based on freshly synced dev.